### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]
Fix tests

File: app/main.py [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    with open("app/config.json", "r") as file:
+
+        file_data = json.load(file)
+
+    fuel_cost = file_data["FUEL_PRICE"]
```
Comment (olena-hul) [inline app/main.py:12]: Use `.get` instead of index access

File: app/customer.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        return total
+
+    def cost_of_the_trip(self, shops: list[Shop], fuel_cost: float) -> None:
+        cheapest_shop = None
+        min_total_cost = float("inf")
+        for shop in shops:
+            spent_for_shopping = round(self.road_cost(shop, fuel_cost) * 2
+                                       + self.product_cost(shop), 2)
+
+            total_trip_cost = spent_for_shopping
```
Comment (olena-hul) [inline app/customer.py:41]: this variable is redundant

**Review by olena-hul — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]
Good job!

**Pull request closed without merging**